Assets/Tests/InputSystem/Plugins/DualShockTests.cs

```diff
@@ -78,6 +78,67 @@ public void Devices_SupportsDualShockAsHID()
         // Sensors not (yet?) supported. Needs figuring out how to interpret the HID data.
     }
 
+    [Test]
+    [Category("Devices")]
+    public void Devices_SupportsDualShock3AsHID()
```

Comment:
Test looks identical? Recommend just using `TestCase` in that case.

Reply:
Differences are in types, not suitable for testcase. But I was able to get rid of the code duplication by making a shared generic method for the test code.